modules/system/oagw/docs/adr-rate-limiting.md

```
@@ -358,10 +358,14 @@ is typically distributed evenly by load balancer.
 **Redis key structure**:
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)